### PR TITLE
Bug 1529340 - CFR Pinned tabs popup animation

### PR DESCRIPTION
**Pull request by rlr**
@piatra this is built on top of the branch from your PR #4824.

The commit includes a patch file for m-c. I'll open the phab review once we think it's stable.

Light theme animating:
screen shot 2019-03-05 at 3 57 07 pm

Light theme paused or with cosmetic animations pref disabled:
screen shot 2019-03-05 at 3 58 08 pm

Dark theme animating:
screen shot 2019-03-05 at 3 58 55 pm

Dark theme paused or with cosmetic animations pref disabled:
screen shot 2019-03-05 at 3 59 30 pm

### CODE REVIEW

File: lib/ActivityStream.jsm [hunk 1]
```diff
@@ -204,7 +204,7 @@ const PREFS_CONFIG = new Map([
   }],
   ["asrouter.devtoolsEnabled", {
     title: "Are the asrouter devtools enabled?",
-    value: false,
+    value: true,
```
Comment (rlr) [inline lib/ActivityStream.jsm]: ignore this. i was just avoiding a trip to `about:config`

**Comment by k88hudson** [comment]
I see some text getting cut off:
image

**Comment by rlr** [comment]
Latest screenshots after rebasing of the latest from #4824 :

screen shot 2019-03-06 at 4 44 04 pm
screen shot 2019-03-06 at 4 44 21 pm

**Comment by rlr** [comment]
The CSS changes are now on phab: https://phabricator.services.mozilla.com/D22398

File: lib/CFRPageActions.jsm [hunk 2]
```diff
@@ -288,34 +274,141 @@ class PageAction {
...
     }
+  }
 
-    footerLink.value = await this.getStrings({string_id: "cfr-doorhanger-extension-learn-more-link"});
-    footerLink.setAttribute("href", content.addon.amo_url);
-    footerLink.onclick = () => this._sendTelemetry({message_id: id, bucket_id: content.bucket_id, event: "LEARN_MORE"});
+  async _renderPinTabAnimation() {
+    const footer = this.window.document.getElementById("cfr-notification-footer");
+    const ANIMATION_CONTAINER_ID = "cfr-notification-footer-pintab-animation-container";
+    let animationsEnabled = Services.prefs.getBoolPref(ANIMATIONS_ENABLED_PREF);
```
Comment (k88hudson) [inline lib/CFRPageActions.jsm]: Do you need to provide a fallback here just in case?

**Comment by k88hudson** [comment]
Thanks, this is looking great now. Can you re-open this against `mozilla/cfr-pinned-tabs` and we can merge it in?

**Comment by rlr** [comment]
closing in favor of pull #4833

**Pull request closed without merging**